scipy/special/basic.py

```diff
@@ -754,20 +754,101 @@ def riccati_yn(n, x):
 
 
 def erfinv(y):
-    """Inverse function for erf.
+    """Inverse of the error function erf.
+
+    Computes the inverse of the error function.
+
+    Inverse of the error function satisfies erf(erfinv(x)) = x.
+
+    Parameters
+    ----------
+    y : ndarray
+        Argument at which to evaluate. Domain: (-1, 1)
```

Comment:
I think you can specify the Domain as the closed interval `[-1, 1]`

Reply:
As far as I know, the domain of the `erf` is [-inf, inf] and the range is (-1, 1).
That means the domain and range of the inverse of `erf` is the opposite case, domain -> range/ range-> domain.
And wiki says that `The inverse error function is usually defined with domain (−1,1)`.

Could you check the domain of the `erf`?

Reply:
Yes, there is the mathematical erf() and the implemented `erf`. The implemented `erf` seems to include `+/I inf` in its domain, and `erfinv` returns it as part of its domain.
```
In [88]: erf([-np.inf, np.inf])
Out[88]: array([-1.,  1.])

In [91]: erfinv(np.array([-1, 1]))
Out[91]: array([-inf,  inf])
```

Reply:
I understand your comment. I just think about `erf` focused on mathematical base.
It seems to be right an implementation point of view.
I will revise this part from an open domain to a closed domain.